README.md

```diff
@@ -7,209 +7,119 @@
+are encoded in a shared data file, and that data is used to fill in
+"Mad Libs"-style templates across each of the tools Magic Modules generates.
 
+They include:
 
-## Motivation
+* Terraform
+* Ansible
+* InSpec
 
-Customers -- large and small -- use many different deployment tools to provision
-and manage their operating environments. Companies like Puppet, Chef, Ansible,
-Hashicorp produce open source tools to meet these needs and typically provide
-enterprise versions and support.
+Not only is Magic Modules a force multiplier for our developers, Magic Modules
+allows us to preemptively solve issues by encoding field-tested learnings about
+the GCP API into each tool; issues solved for one tool will be fixed in each
+other tool.
 
-Once customers invest in these tools they prefer to use a single management
-system (and style). For this reason Google wants to provide integration for
-these tools, so that our customers have an easier path to GCP versus abandoning
-a tool they're already comfortable with. For example Puppet can provision and
-manage VMs in Compute Engine, databases in Cloud SQL, Google Kubernetes Engine
-clusters, and so on.
+## Getting Started with Magic Modules
 
-But given the breadth of products available in Google Cloud Platform, creating
-modules by hand quickly becomes inefficient and costly. It would involve too
-much redundant engineering work and create too much code for a team to maintain.
-There is also the issue of explosion of modules, if you consider for
-comprehensive coverage: <code>
-total\_libraries&nbsp;=&nbsp;total\_products&nbsp;\*&nbsp;total\_providers
-</code>.
-The more products we add to the portfolio, or increase the support to other
-tools, doing this work by hand will not scale.
+Using Magic Modules to make changes to the generated tools might seem
+intimidating, but we're here to help you get familiar with our tooling.
 
-Enter Magic Modules.
+We've prepared an interactive tutorial that you can try out with Open in Cloud
+Shell below; if you're getting set up on a local workstation, this guide serves
+as a reference.
 
+[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/magic-modules&tutorial=TUTORIAL.md)
 
-## Philosophy
+### Requirements
 
-The philosophy behind Magic Modules is a strongly defined object and object
-relationship schema. This schema can in turn be used to map the necessary API
-calls to achieve the object state.
+To get started, you'll need:
 
-> Magic Modules heavily relies on *object convergence* as its target
-> destination. That means that the user will specify the final state they would
-> like their objects (and/or object relations to be) and the code generated by
-> Magic Modules will do "whatever it takes" to make it so.
+* Ruby 2.5.0
+  * You can use `rbenv` to manage your Ruby version(s)
```

Comment:
```suggestion
  * (optional): You can use `rbenv` or `rvm` to install + manage your Ruby installation.
```

Reply:
This is already worded like a suggestion so no need to add optional imo. Also suggesting multiple tools just makes things complicated, let's keep it simple. (I'm on the fence about including this blurb altogether tbh- we shouldn't worry about how users have configured the language on their system)